src/libsync/propagateuploadng.cpp

```diff
@@ -130,32 +130,46 @@ void PropagateUploadFileNG::slotZsyncSeedFailed(QString errorString)
-                                             |
-    +----------------------------------------+
-    |
-    +-> MOVE ------> moveJobFinished() ---> finalize()
-
-
+    +---> doStartUpload()
+          isZsyncPropagationEnabled()?  +--- yes ----> Does remote .zsync metadata file exist?
+             +                                                       +         +
+             |no                                                     |no       |yes
+             |                                                       |         |
+             +^------------------------------------------------------+         |
+             |                                                                 v
+             |                                        Download and seed zsync metadata and set-up new _rangesToUpload
+             |                                                                 +
+             +^----------------------------------------------------------------+
+             v
+          doStartUploadNext()
+          isZsyncPropagationEnabled()?  +--- yes ----> Generate new zsync metadata file
+             +                                                       +
+             |no                                                     |
+             |                                                       |
+             v                                                       |
+          Check the db: is there an entry? <-------------------------+
+             +                           +
+             |no                         |yes
+             |                           v
+             v                        PROPFIND
+             startNewUpload() <-+        +---------------------------+\
+                +               |        +                             \
+               MKCOL            + slotPropfindFinishedWithError()     slotPropfindFinished()
+                +                                                       Is there stale files to remo^e?
+            slotMkColFinished()                                         +                      +
+                +                                                       no                    yes
+                |                                                       +                      +
+                |                                                       |                  DeleteJob
+                |                                                       |                      +
+          +-----+^------------------------------------------------------+^--+  slotDeleteJobFinished()
+          |
+          +---->  startNextChunk()  +-+finished?  +++
+                        ^               +          |
+                        +---------------+          |
+                                                   |
+          +----------------------------------------+
+          |
+          +-> MOVE +-----> mo^eJobFinished() +--> finalize()
```

Comment:
Something happened to lower case 'v', also in "remove" further up

Reply:
Indeed, the ascii visual editor I used seems to have a problem with v's 😟, fixed.